ui/src/app/pages/workspace/workspace-nav-bar.tsx

```diff
@@ -81,6 +104,9 @@ export const WorkspaceNavBarReact = fp.flow(
     {activeTabIndex > 0 && navSeparator}
     {fp.map(tab => navTab(tab, restrictTab(props.workspace, tab)), tabs)}
     <div style={{flexGrow: 1}}/>
+    {workspace && <div data-test-id='cdr-version' style={{textTransform: 'none'}}>
```

Comment:
do we need this check for a workspace if we're already doing that in the parent component before rendering this?

Reply:
That was also my assumption, but workspace is initially undefined here.  I don't know why.

Reply:
This line got dropped somehow.  Re-added in #4191